ci-operator/step-registry/ipi/conf/aws/default/ipi-conf-aws-default-ref.yaml

```diff
@@ -0,0 +1,48 @@
+ref:
+  as: ipi-conf-aws-default
+  from_image:
+    namespace: ocp
+    name: "4.16"
+    tag: upi-installer
+  commands: ipi-conf-aws-default-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: BASE_DOMAIN
+    default: "origin-ci-int-aws.dev.rhcloud.com"
+    documentation: |-
+      A fully-qualified domain or subdomain name. The base domain of the cloud provider is used for setting baseDomain variable of the install configuration of the cluster.
+  - name: COMPUTE_NODE_REPLICAS
+    default: "3"
+    documentation: |-
+      The number of compute/worker nodes to create for the cluster.
+      If SIZE_VARIANT is set to "compact", this is ignored and no workers are created.
+      (default: 3)
+  - name: CONTROL_PLANE_REPLICAS
+    default: "3"
+    documentation: |-
+      The number of control plane nodes to create for the cluster. Useful for SNO cluster provision.
+  - name: OCP_ARCH
+    default: "amd64"
+    documentation: |-
+      The architecture of the control plane nodes (e.g., amd64, arm64).
+  - name: COMPUTE_ARCH
```

Comment:
Seem like we do not need this ENV.

Reply:
Use single `OCP_ARCH` to control master and worker arch?

Reply:
Because whatever `COMPUTE_ARCH` is set, it is always overwritten.

Reply:
Okay, now `OCP_ARCH` controls master and worker arch, I am okay with it.